examples/spring/src/integration-test/kotlin/com/expediagroup/graphql/examples/IntegrationTest.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 Expedia, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.expediagroup.graphql.examples
+
+import com.expediagroup.graphql.examples.Constants.DATA_JSON_PATH
+import com.expediagroup.graphql.examples.Constants.ERRORS_JSON_PATH
+import com.expediagroup.graphql.examples.Constants.EXTENSIONS_JSON_PATH
+import org.springframework.test.web.reactive.server.WebTestClient
+
+interface IntegrationTest {
```

Comment:
there is no need to define this interface, lets rename this file as `WebTestClientExtensions` and move those to top level functions instead

Reply:
I renamed the class, made the functions as top level, and moved the class under `test` directory.